apps/auction/routes.coffee

```diff
@@ -29,10 +29,9 @@ setupUser = (user, auction) ->
           user.set 'registered_to_bid', boolean
         error: ->
           user.set 'registered_to_bid', false
+      ).then( ([ bidder ]) ->
+        user.set 'qualified_for_bidding', bidder?.qualified_for_bidding
```

Comment:
Looking at the full `setupUser`, I'm wondering if there's an error in the original code. I'm guessing the presence of `Q.all` meant that the author was expecting final `setupUser` promise to resolve only after all the async activity is done. I'm guessing user `user.fetch` returns a promise, but without a comma after it, the promise is lost to the ether, and won't delay `setupUser` from resolving.

I'm also curious whether `user.checkRegisteredForAuction` is dependent on `user.fetch`. If so, this should maybe be a sequence of promises, without `Q.all` at all.

Glad to look into this myself if that doesn't make sense, since it probably wasn't your original code.